src/components/foundation/grid/Container.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { ComponentStory } from '@storybook/react';
+
+import Col from './Col';
+import Container from './Container';
+import Row from './Row';
+
+export default {
+  title: 'Grille et points de rupture - Grid System',
```

Comment:
Add tree structure "Composants/..."

Reply:
done, as it's a fundation component, I set it as a `  title: 'Fondamentaux Techniques/Grille et points de rupture - Grid System'`

Reply:
Well done !